src/pages/[platform]/build-a-backend/data/connect-from-server-runtime/nuxtjs-server-runtime/index.mdx

```diff
@@ -0,0 +1,503 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Nuxt.js server runtime',
+  description:
+    'Connect to Amplify Data from Nuxt.js Server-side Runtime (SSR).',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This guide walks through how you can connect to Amplify Data from Nuxt.js Server-side Runtime (SSR). For Nuxt.js applications, Amplify provides first-class support for [Routing (Pages)](https://nuxt.com/docs/getting-started/routing) , [API Routes](https://nuxt.com/docs/guide/directory-structure/server#server-routes) , and [Middleware](https://nuxt.com/docs/guide/directory-structure/server#server-middleware).
+
+Before you begin, you will need:
+
+- [A Nuxt.js application created](https://nuxt.com/docs/getting-started/installation)
+- [Deployed Amplify Data resources](/[platform]/build-a-backend/data/set-up-data/), or directly using [AWS AppSync](https://aws.amazon.com/appsync/)
+
+## Connect to Amplify Data from a Nuxt.js server runtime
+
+Connecting to Amplify Data will include setting up the AmplifyAPIs Plugin with the `runWithAmplifyServerContext` adapter,  using the `useNuxtApp()` composable, setting up the Amplify server context utility and then using the `runAmplifyApi` function to call the API in an isolated server context.
+
+### Step 1 - Set up the AmplifyAPIs Plugin
+
+Nuxt 3 offers universal rendering by default, where your data fetching logic may be executed on both the client and server sides. Amplify offers APIs that are capable of running within a server context to support use cases such as server-side rendering (SSR) and static site generation (SSG), though Amplify's client-side APIs and server-side APIs of Amplify are slightly different. You can set up an AmplifyAPIs plugin to make your data fetching logic run smoothly across the client and server. To learn more about how to use Amplify categories APIs in server side rendering, refer to this [documentation](https://docs.amplify.aws/react/build-a-backend/server-side-rendering/).
```

Comment:
For this link, are you trying to link to gen2 documentation or gen1? If you want to do gen2, you could do `[documentation](/react/build-a-backend/server-side-rendering/)`

Reply:
For now this needs to be a reference to Gen 1 documentation: https://docs.amplify.aws/react/build-a-backend/server-side-rendering/. 

Actually, the next step is to bring the above documentation to a Gen 2 page. We can then update it as you have suggested.